lib/src/Status/OSNotification.mm

```diff
@@ -55,6 +57,17 @@ - (instancetype)initOSNotification:(OSNotification*) instance;
     [notification release];
 }
 
+void OSNotification::showIconBadgeNotificationMacOs(int allMentions)
+{
+    QString mentionsString;
+    if (allMentions > 0 && allMentions < 10) {
+        mentionsString = QString::number(allMentions);
+    } else if (allMentions >= 10) {
+        mentionsString = "9+";
+    }
+    [[NSApp dockTile] setBadgeLabel:mentionsString.toNSString()];
```

Comment:
@endulab in case `mentionsString` is an empty string will this remove badge from the icon? If not then please add `void removeIconBadge()` function.

Reply:
@saledjenic if the string is empty, icon will be removed. I will add a comment about this.